mesonbuild/backend/ninjabackend.py

```diff
@@ -1274,12 +1274,12 @@ def generate_dynamic_link_rules(self, outfile):
                         pass
                 rule = 'rule %s%s_LINKER\n' % (langname, crstr)
                 if mesonlib.is_windows():
-                    command_template = ''' command = {executable} @$out.rsp
+                    command_template = ''' command = {linker_exelist} @$out.rsp
```

Comment:
Oups, my bad, I started to name it {linker_exelist} and renamed it {executable} latter.

I've missed this line in my 'git rebase -i'. I will fix it.